### PR TITLE
fix(components): [row] align="top" is not working properly

### PR DESCRIPTION
**Pull request by yj-liuzepeng**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

## Description

the default value of align-items is 'stretch'

## Related Issue

Fixes #https://github.com/element-plus/element-plus/issues/13538.

### CODE REVIEW

File: packages/components/row/src/row.vue [hunk 1]
```diff
@@ -37,6 +37,6 @@ const style = computed(() => {
 const rowKls = computed(() => [
   ns.b(),
   ns.is(`justify-${props.justify}`, props.justify !== 'start'),
-  ns.is(`align-${props.align}`, props.align !== 'top'),
+  ns.is(`align-${props.align}`),
```
Comment (chenxch) [inline packages/components/row/src/row.vue:40]: Good job, here I have a suggestion, the current change may cause the user's layout to change, my suggestion is to make the following changes:
first setp:
```
ns.is(`align-${props.align}`, props.align),
```
second setp:
Remove the default value of `props.align`.

`justify-start` is currently missing

**Comment by chenxch** [comment]
/cc @YunYouJun Should we not set the default value of `align` like `element ui`?
image

**Comment by YunYouJun** [comment]
Yes, I think we should not set a default value for it.

In addition with the popularity of tailwindcss and unocss today, I think this attribute is no longer needed. 🤣

File: packages/components/row/src/row.vue (hunk 1 above)
Comment (yj-liuzepeng) [reply packages/components/row/src/row.vue]: Thanks, I changed the code，please review @chenxch

**Review by ryuhangyeong — COMMENTED** [review]
## Personal Opinion

In my personal opinion, I think it's better to keep the current state because there is no difference in behavior before and after the code change. 🤔

(I think it will have an impact if the `align-items: flex-start;` property is included, which does not already exist.)

**Comment by yj-liuzepeng** [comment]
> ## Personal Opinion
> In my personal opinion, I think it's better to keep the current state because there is no difference in behavior before and after the code change. 🤔
> 
> (I think it will have an impact if the `align-items: flex-start;` property is included, which does not already exist.)

When setting `align="top"`, the effect is different, the previous setting did not take effect. 
`align-items: flex-start;` has been added in row.scss, this property will take effect,  please take a look

**Review by ryuhangyeong — APPROVED** [review]

**Comment by yj-liuzepeng** [comment]
> > ## Personal Opinion
> > In my personal opinion, I think it's better to keep the current state because there is no difference in behavior before and after the code change. 🤔
> > (I think it will have an impact if the `align-items: flex-start;` property is included, which does not already exist.)
> 
> When setting `align="top"`, the effect is different, the previous setting did not take effect. `align-items: flex-start;` has been added in row.scss, this property will take effect, please take a look

PTAL @chenxch

**Review by btea — APPROVED** [review]

**Comment by btea** [comment]
> Is there anything that does not comply with the regulations, why is it not being merged ？🤔️ @btea

Don't worry, it's fine. I just thought it might take @chenxch  to confirm.

**Pull request merged**